gems/pending/Scvmm/miq_hyperv_disk.rb

```diff
@@ -38,7 +38,10 @@ def open(vm_disk)
     (Get-Item "#{@virtual_disk}").length
 STAT_EOL
     file_size, stderr = @parser.parse_single_powershell_value(run_correct_powershell(stat_script))
-    raise "Unable to obtain virtual disk size for #{vm_disk}" if stderr.include?("At line:")
+    if @network && stderr.include?("RegisterTaskDefinition")
+      raise "Unable to obtain virtual disk size for #{vm_disk}. Check Hyper-V Host Domain Credentials"
+    end
+    raise "Unable to obtain virtual disk size for #{vm_disk}" unless stderr.empty?
```

Comment:
@roliveri it turned out that requiring "At line" was indeed too restrictive.  Powershell also returns Exception errors (as in the specific case I'm checking) that don't include the "At line:" string.  What I can do is check for *either* "At line:" or "Exception calling" but I'm more concerned that there are other error cases that won't be handled if I do so.
@djberg96 do you have experience with other types of errors or cases were STDERR is set on return from winrm that is actually not a failure?